pantos/client/library/blockchains/ethereum.py

```diff
@@ -19,6 +21,13 @@
 from pantos.client.library.blockchains.base import BlockchainClientError
 from pantos.client.library.constants import TOKEN_SYMBOL_PAN
 
+_CONTRACTS_VERSION = semantic_version.Version('1.0.0')
+
```

Comment:
Current version is 1.1.2 just in case.

Reply:
https://github.com/pantos-io/client-library/pull/19#discussion_r1668136621